notify/webhook/webhook.go

```diff
@@ -106,5 +107,16 @@ func (n *Notifier) Notify(ctx context.Context, alerts ...*types.Alert) (bool, er
 	}
 	notify.Drain(resp)
 
-	return n.retrier.Check(resp.StatusCode, nil)
+	return n.retrier.Check(resp.StatusCode, resp.Body)
+}
+
+func errDetails(body io.Reader, url string) string {
+	if body == nil {
+		return url
+	}
+	bs, err := io.ReadAll(body)
+	if err != nil {
+		return url
+	}
+	return fmt.Sprintf("%s at %s", string(bs), url)
```

Comment:
This would be more idiomatic/readable IMHO.

```suggestion
	return fmt.Sprintf("%s: %s", url, string(bs))
```